### ISSUE TITLE
unknown pseudo-class selector ':scope *', when multiple selectors

### ISSUE THREAD

**Issue by ferllings** [jsdom/jsdom#3141]
### Basic info:

- **Node.js version:** v15.11.0
- **jsdom version:** v16.5.1

### Minimal reproduction case

```js
const { JSDOM } = require('jsdom');

(async () => {
  const dom = await new JSDOM('<a><img /><div role="img"></div></a>');

  const a = dom.window.document.body.querySelector('a')
  const childs = a.querySelectorAll(':scope img, :scope div')
  console.log(childs.length)
})();
```
```
The following code generates an error
/node_modules/nwsapi/src/nwsapi.js:565
          err = new global.DOMException(message, 'SyntaxError');
                ^
DOMException [SyntaxError]: unknown pseudo-class selector ':scope *'
    at emit (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/nwsapi/src/nwsapi.js:565:17)
    at compileSelector (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/nwsapi/src/nwsapi.js:1292:17)
    at compile (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/nwsapi/src/nwsapi.js:753:16)
    at collect (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/nwsapi/src/nwsapi.js:1553:24)
    at Object._querySelectorAll [as select] (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/nwsapi/src/nwsapi.js:1518:36)
    at HTMLAnchorElementImpl.querySelectorAll (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/jsdom/lib/jsdom/living/nodes/ParentNode-impl.js:72:26)
    at HTMLAnchorElement.querySelectorAll (/Users/cedric/DocumentsPro/ARC.RulesEngine/node_modules/jsdom/lib/jsdom/living/generated/Element.js:1045:58)
    at /Users/cedric/DocumentsPro/ARC.RulesEngine/test.js:7:20
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
```

The code works fine if the css selector is changed to ':scope img, div' or 'img, :scope div'

### How does similar code behave in browsers?

No error

**Comment by ferllings** [comment]
I tracked down the issue to a nwsapi issue: https://github.com/dperini/nwsapi/issues/33

**Comment by RS-Sautter** [comment]
it seems to me that this issue is fixed in nwsapi, can you please update the dependencies?

**Comment by domenic** [comment]
jsdom's package.json has nwsapi as `^2.2.0`, and so if this is fixed in the current (2.2.1) version of nwsapi, then any new installation of jsdom will get the 2.2.1 version and thus the fix. There's no need to do any manual updates.
